pkgs/development/libraries/sentencepiece/default.nix

```diff
@@ -1,9 +1,9 @@
-{ lib
+{ stdenv
 , fetchFromGitHub
-, stdenv
 , cmake
 , gperftools
 
+, enableStatic ? false
```

Comment:
there should be a related:
```
  sentencepiece = super.sentencepiece.override {
    enableStatic =  true;
  };
```
in `pkgs/top-level/static.nix`

Reply:
Thanks! I have added this.

I also added a commit which puts `gperftools` in `buildInputs` rather than `nativeBuildInputs`.

However, building `sentencepiece` statically fails, because building `gperftools` fails with:

```
libtool: link: x86_64-unknown-linux-musl-g++ -std=gnu++11 -Wall -Wwrite-strings -Woverloaded-virtual -Wno-sign-compare -Wno-unused-result -fsized-deallocation -faligned-new -DNO_FRAME_POINTER -g -O2 -o stacktrace_unittest src/tests/stacktrace_unittest.o  ./.libs/libstacktrace.a ./.libs/liblogging.a ./.libs/libfake_stacktrace_scope.a /nix/store/yzda7vvxr4zpmmlp5bfb22s075dm3dhf-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64/libstdc++.so -lm -Wl,-rpath -Wl,/nix/store/yzda7vvxr4zpmmlp5bfb22s075dm3dhf-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64 -Wl,-rpath -Wl,/nix/store/yzda7vvxr4zpmmlp5bfb22s075dm3dhf-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64
/nix/store/9nh6bp6w99kyjsa1gi6ynd3vxp1x66xl-x86_64-unknown-linux-musl-binutils-2.31.1/bin/x86_64-unknown-linux-musl-ld: attempted static link of dynamic object `/nix/store/yzda7vvxr4zpmmlp5bfb22s075dm3dhf-x86_64-unknown-linux-musl-stage-final-gcc-debug-9.3.0-lib/x86_64-unknown-linux-musl/lib/../lib64/libstdc++.so'
```

Seems to be the same issue as #76873.